code/modules/virus2/effect.dm

```diff
@@ -70,7 +70,7 @@
 	chance_maxm = 0
 
 /datum/disease2/effect/gibbingtons
-	name = "Gibbingtons Syndrome"
+	name = "Gibbington's Syndrome"
 	stage = 4
 	badness = 3
 	activate(var/mob/living/carbon/mob,var/multiplier)
```

Comment:
Can you make all of this use absolute pathing while you're in the file?

Reply:
I think I'm totally missing what you're asking here. Which paths aren't absolute?

Reply:
activate() is a proc definition and should be `/datum/disease2/effect/gibbingtons/activate(var/mob/living/carbon/mob,var/multiplier)`

Reply:
And it's like that for just about every proc in the file, I just noticed it while reviewing the diff and figured it'd be easier to make one comment than to mark every single instance of relative pathing